spec-examples/src/blake2/blake2b.rs

```diff
@@ -1,12 +1,14 @@
 // Import hacspec and all needed definitions.
 use hacspec::prelude::*;
 
-array!(State, 8, U64);
-array!(DoubleState, 16, U64);
-array!(Counter, 2, u64);
-bytes!(Buffer, 128);
-bytes!(Digest, 64);
+array!(StateB, 8, U64);
```

Comment:
Could we avoid having `StateB` and `StateS` in addition to `State`?

Reply:
Yes we can ! Did it